packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 ## Unreleased
 
+* Changed
+ * Updated Custom Icon + message + buttons example for `terra-status-view`.
+ * Updated `terra-status-view` Change Variant example component from HTML Native select to `terra-form-select` component.
```

Comment:
```suggestion
 * Updated `terra-status-view - Change Variant` example component to use the `terra-form-select` component instead of HTML native `<select>`.
```

Reply:
changes resolved.